### ISSUE TITLE
Hyperparameter Tuning

### ISSUE THREAD

**Issue by emiklap** [MalihaT111/ai-recruitment#12]
Optimize learning rate, batch size, and number of epochs

Model Evaluation: Use metrics like F1-score, accuracy, and confusion matrix.

Assign specific tasks for error analysis 
(different people are assigned different preprocessing steps to adjust, which we will all bring together on October 25)
